packages/fuel-gauge/src/script-with-vectors.test.ts

```diff
@@ -39,21 +39,15 @@ describe('Script With Vectors', () => {
     const formattedLog = logs.map((l) => (typeof l === 'string' ? l : l.toNumber()));
 
     const vecFirst = someVec[0];
-    const vecPointer = 11232;
     const vecCapacity = 4;
     const vecLen = 4;
-    const vecAddr = 11208;
 
     expect(formattedLog).toEqual([
       vecFirst,
-      'vector.ptr()',
```

Comment:
`raw_ptr` encoding has been [intentionally left out](https://github.com/FuelLabs/sway/pull/5848#pullrequestreview-1997167951) of the experimental build.